.stickler.yml

```diff
@@ -0,0 +1,10 @@
+linters:
+    csslint: {  }
+    stylelint:
+        config: 'stylelint.config.js'
+files:
+    ignore:
+      - 'public/*'
+      - '*.log'
+      - 'package.json'
+      - 'reset.css'
```

Comment:
Please do not ignore files in your stickler config as it is essentially the same as not using stickler at all. Up until recently, I thought stickler was an unnecessary addition to every code base, however, after using it for a while, I have found that it is one of the best tools to ever be created as it has taught me to write more standardized code and also on a more recent project, it saved me hours of time in debugging something I would have never caught with my mere eyes. I hope you come to see the stickler as I have, please give it some strong thought as it is a hard requirement for this project :smile: 
![screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png)

Reply:
Okay, thank you for the review!